definitions/ext-datadomain/definition.yml

```diff
@@ -0,0 +1,19 @@
+domain: EXT
+type: DATADOMAIN
+
+synthesis:
+  rules:
+  - identifier: clusterCI
```

Comment:
Hi 👋 

I wondered if this identifier is unique within a customer account ?
E.g. other entity types concatenate several fields to guarantee uniqueness: https://github.com/newrelic/entity-definitions/blob/962e0c8363b0b963cc8b209547f361c3d186591c/definitions/ext-pixie_kafkabroker/definition.yml#L5

Reply:
Yes, it is unique.